index.js

```diff
@@ -91,7 +106,7 @@ const promo20 = users => {
 // ***************************************************************************
 
 const addActive = users => {
-  // Your code goes here ...
+  users.forEach(user => user.isActive = true)
```

Comment:
This function immediately is changing the values you are passing onto it.

In functions, unless said otherwise, try to create a habit that you return a totally new value.
For example
```js
const addActive = users => {
return users.map(user => {
return {
...user, //here you are getting the whole user object copied
isActive: true
}
})
}
```
After this function you get an array of every user active but you get a totally new value.